rbac/management/principal/cleaner.py

```
@@ -245,6 +355,179 @@ def process_umb_event(frame, umb_client: Stomp, bootstrap_service: TenantBootstr
+
+            else:
+                # NORMAL MODE: Actually update the database
+                # By default, only process disabled users.
+                # If the setting is enabled, process all users.
+                if not user.is_active or settings.PRINCIPAL_CLEANUP_UPDATE_ENABLED_KAFKA:
+                    # If Tenant is not already ready, don't ready it
+                    bootstrap_service.update_user(user, ready_tenant=False)
+
+                kafka_messages_success_total.inc()
+                return MessageProcessingResult(should_continue=True, success=True)
+        except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)